app/queries/get_courses_not_yet_open_for_application.rb

```diff
@@ -0,0 +1,11 @@
+class GetCoursesNotYetOpenForApplication
+  attr_reader :application_form
+
+  def initialize(application_form:)
+    @application_form = application_form
+  end
+
+  def call
+    application_form.application_choices.select { |choice| choice.course.open_for_applications? }
```

Comment:
```suggestion
    application_form.application_choices.select { |choice| choice.course.open_for_applications? }.map(&:course)
```

so that you return courses, not app choices